CapabilityService.sln

```diff
@@ -0,0 +1,37 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.28803.202
+MinimumVisualStudioVersion = 15.0.26124.0
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CapabilityService.Tests", "src\CapabilityService.Tests\CapabilityService.Tests.csproj", "{24DDC001-487E-49D7-A783-C69939C8B759}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "CapabilityService.WebApi", "src\CapabilityService.WebApi\CapabilityService.WebApi.csproj", "{578832AA-9641-4789-A65F-B21E29701FB1}"
+EndProject
+Project("{E53339B2-1760-4266-BCC7-CA923CBCF16C}") = "docker-compose", "docker-compose.dcproj", "{DA41CF6F-E45B-40B5-9BCF-D3259358B4F0}"
+EndProject
+Global
+	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
+		Release|Any CPU = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(ProjectConfigurationPlatforms) = postSolution
+		{24DDC001-487E-49D7-A783-C69939C8B759}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{24DDC001-487E-49D7-A783-C69939C8B759}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{24DDC001-487E-49D7-A783-C69939C8B759}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{24DDC001-487E-49D7-A783-C69939C8B759}.Release|Any CPU.Build.0 = Release|Any CPU
+		{578832AA-9641-4789-A65F-B21E29701FB1}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{578832AA-9641-4789-A65F-B21E29701FB1}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{578832AA-9641-4789-A65F-B21E29701FB1}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{578832AA-9641-4789-A65F-B21E29701FB1}.Release|Any CPU.Build.0 = Release|Any CPU
+		{DA41CF6F-E45B-40B5-9BCF-D3259358B4F0}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{DA41CF6F-E45B-40B5-9BCF-D3259358B4F0}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{DA41CF6F-E45B-40B5-9BCF-D3259358B4F0}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{DA41CF6F-E45B-40B5-9BCF-D3259358B4F0}.Release|Any CPU.Build.0 = Release|Any CPU
+	EndGlobalSection
+	GlobalSection(SolutionProperties) = preSolution
+		HideSolutionNode = FALSE
+	EndGlobalSection
+	GlobalSection(ExtensibilityGlobals) = postSolution
+		SolutionGuid = {E4061A38-92D4-409F-ABB0-1FEF67EAF051}
+	EndGlobalSection
+EndGlobal
```

Comment:
Why did this file move?

Reply:
If this file is located in the src folder, and not the root of your repo, then there is no way to make dotnet CLI restore packages from other projects in your solution without dying. Its an odd bug, but if you try moving the sln into the src folder you will notice that it breaks even thou the container filesystem technically should be correct. My guess is that its some dotnet CLI internals that doesnt resolve correctly, but I have never bothered figuring out how to work around it because I always put my .sln files at the root of my repos. :)